README.md

```diff
@@ -22,29 +22,31 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
 
 This integration doesn't rely on a hardcoded list of devices anymore, but relies on the characteristics of every device. This means that more devices will be supported out of the box, based on their category, available states and commands. If your device is not supported or working correctly, have a look [here](#device-not-supported--working-correctly).
 
-## TaHoma services
-
-After installation this integration adds new services to Home Assistant which can be used in automations. The new services are:
-
-+ Somfy TaHoma: Set cover position with low speed (tahoma.set_cover_position_low_speed, only added if supported by the device)
-+ Somfy TaHoma: My position (cover) (tahoma.set_cover_my_position)
-+ Somfy TaHoma: Execute command (tahoma.execute_command)
-+ Somfy TaHoma: Get execution history (tahoma.get_execution_history)
-
-If you want to move your covers always with low speed create [templates](https://www.home-assistant.io/integrations/cover.template/) for each cover which calls the first service.
-
 ## Installation
 
 ### Manual
 
-Copy the `custom_components/tahoma` to your `custom_components` folder. Reboot Home Assistant and install the Somfy TaHoma integration via the integrations page.
+Copy the `custom_components/tahoma` to your `custom_components` folder. Reboot Home Assistant and install the Somfy TaHoma integration via the integrations page or press the blue button below.
 
 ### HACS
 
-This integration is included in HACS. Search for the `Somfy TaHoma` integration and choose install. Reboot Home Assistant and install the Tahoma integration via the integrations page.
+This integration is included in HACS. Search for the `Somfy TaHoma` integration and choose install. Reboot Home Assistant and install the Somfy TaHoma integration via the integrations page or press the blue button below.
```

Comment:
Would be nice to move this at the top, to make the button more visible.

Reply:
What do you mean? The blue button only works after you did install the integration manually or via HACS and after a reboot. Thus I believe we should not list it earlier than our install instructions.

Reply:
What happens when you click the button without the component installed? I tend to agree with @tetienne that it would be nice to have it on top, even if it doesn't work when not installed.

Reply:
My bad I misread the button. I thought it was to install our component.

Reply:
> What happens when you click the button without the component installed? I tend to agree with @tetienne that it would be nice to have it on top, even if it doesn't work when not installed.

It will just give an error (which is not even very nice).